website/versioned_docs/version-0.10.0/ergo.md

```diff
@@ -29,7 +29,7 @@ To achieve those goals the design of Ergo is based on the following
 principles:
 
 - Ergo contracts have a class-like structure with clauses akin to methods
-- Ergo can handle types (concepts, transations, etc) defined with the [Composer Concerto Modeling Language](https://github.com/hyperledger/composer-concerto) (so called CML models), as mandated by the Accord Project Template Specification
+- Ergo can handle types (concepts, transations, etc) defined with the [Composer Concerto Modeling Language](https://github.com/accordproject/concerto) (so called CML models), as mandated by the Accord Project Template Specification
```

Comment:
This is a static versioned file and shouldn't be edited, please remove this change.

When we make a new release of the documentation, Docusarus creates these `versions_docs` automatically as a snapshot of the documentation for historical reference.